README.md

```diff
@@ -31,7 +31,12 @@ swap(one, -1, 0)
 console.log(one) // [1]
 
 function swap(array, first_index, second_index) {
-  // -- YOUR ANSWER HERE --
+  if (array.length > 1) {
+    let tmp1 = array[first_index];
+    let tmp2 = array[second_index];
+    array.splice(first_index, 1, tmp2);
+    array.splice(second_index, 1, tmp1);
+  }
```

Comment:
Good, but do we need to use `splice` or can we use a more basic feature to do this?